### DISCUSSION TITLE
How do I get Binance data into proper Quote format (in Python)?

### DISCUSSION THREAD

**Discussion by DaveSkender** [DaveSkender/Stock.Indicators#781] [Q&A] [answered]
Question from a user.  They're getting quote data from Binance okay, but are struggling to get it into the [appropriate format](https://daveskender.github.io/Stock.Indicators.Python/guide/#historical-quotes) for the Python wrapper.  Here's what they have so far, an array of Quote:

![image](https://user-images.githubusercontent.com/8432125/164994236-f9ae01ca-582b-4791-8c3a-f4bf272693e2.png)

**Comment by LeeDongGeon1996** [comment, marked as answer]
I think this snippet would work!

**Comment by DaveSkender** [comment]
> [!TIP]
> For a working example using price quotes from a **pandas.DataFrame**, see our [_Replit_ of the Williams Fractal indicator](https://replit.com/@daveskender/Stock-Indicators-for-Python-Williams-Fractal).  If you're having general trouble getting setup, see our [QuickStart guide](https://github.com/DaveSkender/Stock.Indicators.Python.QuickStart#readme).

See also:

- #1165
